2-mandatory/3-bush-berries.js

```diff
@@ -21,8 +21,16 @@
   Let's first look at an example that will teach you how to use these methods.
 */
 
+function toCheckAllBerriesSafe(num) {
+  return num.includes("pink");
+}
+
 function isBushSafe(berryArray) {
-  //Write your code here
+  if (berryArray.every(toCheckAllBerriesSafe)) {
+    return "Bush is safe to eat from";
+  } else {
+    return "Toxic! Leave bush alone!";
+  }
 }
```

Comment:
good use of .every method here. The naming of your param `num` is a bit confusing, something like `colour` might work better, then in your function you can do `return colour === "pink"`